### ISSUE TITLE
[Question] About the internationalization of Ledger Live

### ISSUE THREAD

**Issue by clintnetwork** [LedgerHQ/ledger-live-desktop#2075]
Hey,

On Ledger Live i've `Display Languag` and nothing to select, only `Use system language` does it's a problem or it's normal, do you plan to propose translations ?

Thank you

**Comment by gre** [comment]
Hi,

Ledger Live is been translated in 6 languages at the moment, the translations are already available for test: you need to go to `Settings > Experimental features` and toggle on `Translation testing`.

At the moment we are still focusing on these 6 languages and soon will have a better process to explain. Basically we are using Crowdin, an open platform, anyone can contribute to translation but only proofreaders can validate them in. https://crowdin.com/project/ledger-wallet

We are still not yet sure how we will scale with the validation so in the meantime we validate the 6 languages ourself. Soon we'll have more documentation and process :)

**Comment by clintnetwork** [comment]
Yeah just after posting this issue, I found this experimental feature ^^ 

I will participate to the french translation :D

**Comment by gre** [comment]
feel free to contribute on crowdin / comments & suggest better translations :) 🚀

**Comment by ElvisPresly** [comment]
The Cyrillic font is just terrible, can it be fixed? Russian

**Comment by gre** [comment]
@ElvisPresly can you give more detail? here is what I get on my side and I don't see what is the issue.

<img width="1136" alt="Capture d’écran 2019-07-17 à 15 09 17" src="https://user-images.githubusercontent.com/211411/61378453-b918ef00-a8a5-11e9-929b-e1b107ddbfec.png">

thanks!

**Comment by ElvisPresly** [comment]
@gre I do not know as to explain it. But Font ugly. It is Fatter and angular than English font
<img width="1136" alt="61378453-b918ef00-a8a5-11e9-929b-e1b107ddbfec" src="https://user-images.githubusercontent.com/18265958/61379499-56792080-a8b0-11e9-963c-8dea778f375b.png">
